src/App.tsx

```diff
@@ -1,14 +1,40 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useMemo,
+  useState,
+} from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { DropDownItem } from './components/DropDownItem';
+import { useDebounce } from './hooks/useDebounce';
+import { searchedPeople } from './helpers';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [query, setQuery] = useState('');
+
+  const debouncedQuery = useDebounce(query, 1000);
+
+  const onSelect = useCallback(
```

Comment:
```suggestion
  const handleSelect= useCallback(
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364